docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -36,130 +41,78 @@ log data has the following data retention guidelines:
 
-*** Enable *Move to warm phase on rollover*. The index might move to the warm phase
-more quickly than intended if it reaches the *Maximum index size* before the
-the *Maximum age*.
+.. Set *Move data into phase when* to *30 days*. This keeps indices in the hot
+tier for 30 days. After 30 days, indices move to the warm tier.
 
-.. In the *Select a node attribute to control shard allocation* dropdown, select
-*data:warm(2)* to migrate shards to warm data nodes.
+.. Enable **Replicas** and change *Number of replicas* to *1*.
 
-.. Change *Number of replicas* to *1*.
-
-.. Enable *Force merge data* and set *Number of segments* to *1*.
-
-NOTE:  When rollover is enabled in the hot phase, action timing in the other phases
-is based on the rollover date.
+.. Enable *Force merge* and set *Number of segments* to *1*.
 --
 
-. Activate the delete phase and set *Timing for delete phase* to *90* days.
+. Activate the delete phase and set *Move data into phase when* to *90* days.
 +
 [role="screenshot"]
-image::images/ilm/tutorial-ilm-delete-rollover.png["Add a delete phase"]
-
-[discrete]
-==== Create a custom policy
+image::images/ilm/tutorial-ilm-delete-rollover.png[Add a delete phase]
 
-If meeting a specific retention time period is most important, you can create a
-custom policy.  For this option, you use {filebeat} daily indices without
-rollover.
-
-To create a custom policy:
-
-. Go to Management and select *Index Lifecycle Policies*.
-. Click *Create policy*.
-. Activate the warm phase and configure it as follows:
-+
---
-**Timing for warm phase**: 30 days from index creation
-
-**Node attribute**: `data:warm`
-
-**Number of replicas**: 1
-
-**Force merge data**: enable
-
-**Number of segments**: 1
-
-[role="screenshot"]
-image::images/ilm/tutorial-ilm-custom-policy.png["Modify the custom policy to add a warm phase"]
---
-
-. Activate the delete phase and set the timing to 90 days.
-+
-[role="screenshot"]
-image::images/ilm/tutorial-ilm-delete-phase-creation.png["Delete phase"]
-
-To configure the index to use the new policy:
-
-. Go to Management and select *Index Lifecycle Policies*.
-. Find your {ilm-init} policy and click its *Actions* link.
-. Choose *Add policy to index template*.
-. Select your {filebeat} index template name from the *Index template* list. For example, `filebeat-7.5.x`.
-. Click *Add Policy* to save the changes.
-+
-NOTE: If you initially used the default {filebeat} {ilm-init} policy, you will
-see a notice that the template already has a policy associated with it. Confirm
-that you want to overwrite that configuration.
-
-When you change the policy associated with the index template, the active
-index will continue to use the policy it was associated with at index creation
-unless you manually update it. The next new index will use the updated policy.
-For more reasons that your {ilm-init} policy changes might be delayed, see
-<<update-lifecycle-policy, Update Lifecycle Policy>>.
```

Comment:
I removed this section entirely. I'm not really sure if there's an advantage to creating and applying a new policy. Updating the built-in index templates is also tricky.

Reply:
Yeah, this doesn't look like something we want to encourage.